intellij-idea/.idea/runConfigurations/logstash_apm___WebRestApiApplication.xml

```diff
@@ -0,0 +1,14 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="logstash_apm - WebRestApiApplication" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+    <module name="metasfresh-webui-api" />
+    <option name="SPRING_BOOT_MAIN_CLASS" value="de.metas.ui.web.WebRestApiApplication" />
+    <option name="VM_PARAMETERS" value="-Xmx736M -DPropertyFile=$ProjectFileDir$\metasfresh.properties -Dspring.profiles.active=logstash -javaagent:$ProjectFileDir$/elastic-apm-agent-1.16.0.jar -Delastic.apm.service_name=webapi -Delastic.apm.server_url=http://localhost:8200 -Delastic.apm.application_packages=de.metas.*,org.adempiere.*,org.compiere.* -Delastic.apm.enable_log_correlation=true" />
```

Comment:
I believe we should strive to have less custom params in our launch configs. 
Is there no way to have all of these flags (ie. `-Delastic.apm.application_packages=de.metas.*,org.adempiere.*,org.compiere.* -Delastic.apm.enable_log_correlation=true`) in onemofmthe config files?

Reply:
true. AFAIR we can put it into some sort of properties file

Reply:
Remains open issue: https://github.com/metasfresh/me03/issues/4822#issuecomment-637964538